data-prepper-plugins/grok-prepper/src/test/java/com/amazon/dataprepper/plugins/prepper/grok/GrokPrepperTests.java

```diff
@@ -135,6 +175,11 @@ public void testMatchMerge() throws JsonProcessingException {
         assertThat(grokkedRecords.size(), equalTo(1));
         assertThat(grokkedRecords.get(0), notNullValue());
         assertThat(equalRecords(grokkedRecords.get(0), resultRecord), equalTo(true));
+        verify(grokProcessingMatchSuccessCounter, times(1)).increment();
+        verify(grokProcessingErrorsCounter, never()).increment();
```

Comment:
This is very thorough and I like it a lot!

You can do something similar using `verifyNoInteractions`, and it can take multiple mocks:

```
verifyNoInteractions(grokProcessingErrorsCounter, grokProcessingMatchFailureCounter, grokProcessingTimeoutsCounter);
```

But, I leave it up to you.